clients/unshaded/src/main/java/tachyon/client/ClientOptions.java

```diff
@@ -35,8 +34,8 @@
     private TachyonStorageType mTachyonStorageType;
     /** How this operation should interact with the under storage */
     private UnderStorageType mUnderStorageType;
-    /** Worker location to execute this operation, if not possible, the operation will fail */
-    private NetAddress mLocation;
+    /** Worker location to write data, if not possible, the operation will fail */
```

Comment:
Yeah, would rather do it in a separate PR.  It is less clear to me what it means to specify a location for reads though, since the specified location might not have the data we are looking for at all?